public/js/wsClient.js

```diff
@@ -0,0 +1,16 @@
+const { io } = require("socket.io-client");
+const {logger} = require('./logger.js');
+const port = process.env.PORT || 3001;
+const socket = io(`ws://localhost:${port}`);
```

Comment:
consider to change the address of the socket io to the dotenv and choose an address less local @shirbalas